### DISCUSSION TITLE
how to change quality height on HLS(m3u8) on react native like

### DISCUSSION THREAD

**Discussion by Tech2cool** [TheWidlarzGroup/react-native-video#3544] [Q&A]
### What are you trying to do?

I was trying to find a way to change the quality height on an HLS (m3u8) playlist with react-native-video, but I didn't find any option or documentation about it.

### Relevant code

```tsx
"react-native-video": "^6.0.0-beta.5",
```

### Version

v6

**Comment by freeboub** [comment]
This question is not clear to me. you want to force selection of a video track ?
